dotnet/src/SemanticKernel.Core/Functions/NativeFunction.cs

```diff
@@ -517,7 +517,8 @@ Task<FunctionResult> Function(ITextCompletion? text, AIRequestSettings? requestS
             var parameterView = new ParameterView(
                 name,
                 parameter.GetCustomAttribute<DescriptionAttribute>(inherit: true)?.Description ?? string.Empty,
-                defaultValue?.ToString() ?? string.Empty);
+                defaultValue?.ToString() ?? string.Empty,
```

Comment:
Do we need to do anything on 377 re SKFunctionAttribute?

Reply:
SKParameterAttribute is being removed (#2991)